app/src/main/java/com/breadwallet/entities/IntroLanguageResource.kt

```diff
@@ -3,7 +3,10 @@ package com.breadwallet.entities
 import com.breadwallet.R
 
 class IntroLanguageResource {
-
```

Comment:
Is this something destroyed because the latest work lost this @josikie

Reply:
It didn't lost. The lost are the code for FCM. It was already merged before. I saw that the pusher set up still there.